politeiad/backendv2/tstorebe/plugins/pi/cmds.go

```diff
@@ -33,6 +33,29 @@ const (
 	dataDescriptorBillingStatus = pluginID + "-billingstatus-v1"
 )
 
+var (
+	// billingStatusChanges contains the allowed billing status transitions. If
+	// billingStatusChanges[currentStatus][newStatus] exists then the the billing
+	// status transition is allowed.
+	billingStatusChanges = map[pi.BillingStatusT]map[pi.BillingStatusT]struct{}{
+		// Active to...
+		pi.BillingStatusActive: {
+			pi.BillingStatusClosed:    {},
+			pi.BillingStatusCompleted: {},
+		},
+		// Closed to...
+		pi.BillingStatusClosed: {
+			pi.BillingStatusActive:    {},
+			pi.BillingStatusCompleted: {},
+		},
+		// Completed to...
+		pi.BillingStatusCompleted: {
+			pi.BillingStatusActive: {},
+			pi.BillingStatusClosed: {},
+		},
+	}
+)
+
 // cmdSetBillingStatus sets proposal's billing status.
 func (p *piPlugin) cmdSetBillingStatus(token []byte, payload string) (string, error) {
```

Comment:
A formatting nit that I'm just noticing, can you list the cmds/methods that are related to one another next to each other in the plugin API spec and in the cmd implemenations?

```
	// CmdSetBillingStatus command sets the billing status.
	CmdSetBillingStatus = "setbillingstatus"

	// CmdSummary command returns a summary for a proposal.
	CmdSummary = "summary"

	// CmdBillingStatusChanges command returns the billing status changes
	// of a proposal.
	CmdBillingStatusChanges = "billingstatuschanges"
```

i.e. the billing status commands should be listed next to each other in `plugins/pi.go` and the cmd implemenations should be listed in the same order in `pi/cmds.go`.